ui/src/views/Rules/Rule/index.jsx

```diff
@@ -401,6 +448,19 @@ function Rule({ isNewRule, user, ...props }) {
       {error && <ErrorPanel fixed error={error} />}
       {!isLoading && (
         <Fragment>
+          <div>
+            <p className={classes.signoffLabel}>
+              Required Signoffs:
```

Comment:
Let's use a more complete sentence here: "Requires signoff from:"

This will also necessitate a different string when no signoffs are required. (Right now "None" is used, let's change that to "Nobody".)

Reply:
Will work on this